src/main/kotlin/net/maizegenetics/phgv2/brapi/model/Context.kt

```diff
@@ -1,4 +1,15 @@
+ *
+ * OpenAPI spec version: 2.0
+ * 
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+package model
 
 import kotlinx.serialization.Serializable
```

Comment:
Could this class be turned into a simple Data Class?  I am not clear the need for this.  Should probably clean up the above comment as well as having HTML in the comment is not readable in raw from.

Reply:
This was  generated by Swagger Hub.  It is required as several of the response models ( SampleListResponse, ServerInfoResponse, VariantsResponse) have this in their class definitions.

```
data class SampleListResponse(
    val metadata: Metadata,
    val result: SampleListResponseResult,
    val atcontext: Context? = null
)
```

For now, I'd like to leave it alone

Reply:
@zrm22 If this were to be declared as :

typealias Context ArrayList<String?>

Where would that live?

Reply:
@zrm22 Html is gone, this is reverted to what it was when Terry first added it.  My fault for inadvertently changing the package name.